components/KiboConfirmOrder.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="sf-confirm-order" data-testid="confirm-order">
+    <SfHeading
+      :title="$t('Order Details')"
+      :level="2"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <KiboOrderLineItems :order="order" />
+
+    <hr class="hr-order" />
+    <div class="order-total">
+      <SfProperty
+        :name="$t('Subtotal')"
+        :value="$n(order.subtotal, 'currency')"
+        class="sf-property--full-width sf-confirm-order__property sf-confirm-order__property-subtotal"
+      >
+      </SfProperty>
+      <SfProperty
+        :name="$t('Shipping')"
+        :value="$n(order.shippingTotal, 'currency')"
+        class="sf-property--full-width sf-confirm-order__property"
+      >
+      </SfProperty>
+      <SfProperty
+        :value="$n(order.taxTotal, 'currency')"
+        class="sf-property--full-width sf-confirm-order__property"
+      >
+        <template #name>
+          <div class="sf-property__name">
+            {{ $t("Estimated Tax") }}
+            <SfBadge class="sf-badge--number sf-badge color-secondary tax-badge">i</SfBadge>
+          </div>
+        </template>
+      </SfProperty>
+      <SfDivider class="sf-confirm-order__divider" />
+      <SfProperty
+        :name="$t('Total Price')"
+        :value="$n(order.total, 'currency')"
```

Comment:
use KiboPrice using slots

Reply:
Will integrate later once it becomes available in develop branch from your PR.